src/components/MaterialUI/Feedback/NotificationContext.js

```diff
@@ -0,0 +1,20 @@
+import React, { createContext, useState } from "react";
+import { useNotification } from "./useNotification";
+import Notification from "./../DataDisplay/Notification";
+
+export const NotificationContext = createContext();
+
+export const NotificationContextProvider = ({ children }) => {
+	const [snackPack, setSnackPack] = useState([]);
+
+	const addNotification = useNotification(snackPack, setSnackPack);
+
+	return (
+		<NotificationContext.Provider value={{ addNotification }}>
```

Comment:
Just curious, why the "."?

Reply:
createContext creates object with 2 main properties - Provider and Consumer. Depends on your need you should use one of those 2